documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -6,22 +6,14 @@
 [id="about-cold-warm-migration_{context}"]
 = About cold and warm migration
 
-{project-short} supports cold migration from {rhv-full} ({rhv-short}), {ocp-short}, {osp}, VMware vSphere, and Open Virtial Application files that originate from or are compatible with VMware vSphere.
+{project-short} supports cold migration from:
 
-{project-short} supports warm migration from VMware vSphere and from {rhv-short}.
-
-Migration using Open Virtual Application (OVA) files that were created by or are compatible with VMware vSphere as source providers is a Technology Preview.
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
```

Comment:
why we decided to remove OVA from this list?

Reply:
@ahadas asked me to remove it -- even though users can create an OVA source provider.

Reply:
right, because it's hard to describe it as a cold or warm migration - it's definitely not warm migration but also not exactly cold migration..

Reply:
ack, thanks for clarifying.